Simplenote/StorageSettings.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+@objcMembers
+class StorageSettings: NSObject {
+
+    /// URL for the managed object model resource
+    ///
+    let modelURL = Bundle.main.url(forResource: Constants.resourceName, withExtension: Constants.resourceType)!
+
+    /// In app core data storage URL
+    ///
+    let legacyStorageURL = FileManager.default.documentsURL.appendingPathComponent(Constants.sqlFile)
```

Comment:
We should probably keep:

```
private let fileManager = FileManager.default`
```

That would allow us to do dependency injection, and test all the things